region/maridia/inner-yellow.json

```diff
@@ -3206,7 +3322,139 @@
+                    "h_canArtificialMorphIBJ",
+                    "h_canArtificialMorphSpringBallBombJump",
+                    {"ammo": { "type": "PowerBomb", "count": 1 }},
+                    {"or": [
+                      "h_canArtificialMorphStaggeredIBJ",
+                      {"ammo": { "type": "PowerBomb", "count": 1 }},
+                      {"enemyDamage": { "enemy": "Ripper", "type": "contact", "hits": 1 }}
+                    ]}
+                  ],
+                  "devNote": "canSpringBallBombJump can be bypassed with SpringBall and other requirements, but that's being assumed with Bombs and SpringBall at this level."
+                },
+                {
+                  "name": "G-Mode Morph IBJ, PB",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "h_canArtificialMorphIBJ",
+                    {"or": [
+                      "Gravity",
+                      {"ammo": { "type": "PowerBomb", "count": 1 }},
+                      {"enemyDamage": { "enemy": "Ripper", "type": "contact", "hits": 1 }}
+                    ]},
+                    {"ammo": { "type": "PowerBomb", "count": 1 }},
+                    {"or": [
+                      "h_canArtificialMorphStaggeredIBJ",
+                      {"ammo": { "type": "PowerBomb", "count": 1 }},
+                      {"enemyDamage": { "enemy": "Ripper", "type": "contact", "hits": 1 }}
+                    ]}
+                  ]
+                },
+                {
+                  "name": "G-Mode Morph IBJ, PB, Spring Ball",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "h_canArtificialMorphIBJ",
+                    "h_canArtificialMorphSpringBallBombJump",
+                    {"ammo": { "type": "PowerBomb", "count": 1 }},
+                    {"or": [
+                      "h_canArtificialMorphStaggeredIBJ",
+                      {"ammo": { "type": "PowerBomb", "count": 1 }},
+                      {"enemyDamage": { "enemy": "Ripper", "type": "contact", "hits": 1 }}
+                    ]}
+                  ]
+                },
+                {
+                  "name": "G-Mode Morph IBJ, PB",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "h_canArtificialMorphIBJ",
+                    {"ammo": { "type": "PowerBomb", "count": 1 }},
```

Comment:
1) The `h_canArtificialMorphSpringBallBombJump` strat is exactly this with an extra tech so won't be used.
2) Its hard to place the power bomb and avoid damage.  I could do it with double bomb jumps, or damage boosting on the ripper.  But you also need to know precisely where to put the PB.

Reply:
I figured the spring ball variant was a bit easier and would be chosen first since it was higher in the list - it would have a different title and was considering a different description. I didn't notice they ended up having the exact same requirements. I'll update it to have a description that mentions spring ball instead of being two identical strats.